doc/release-notes/4.0.md

```diff
@@ -59,6 +59,12 @@ In addition to more documentation, several blog posts are planned to explain the
 
 ## Breaking changes from 3.2
 
+> [!CAUTION]
```

Comment:
nit: I think that it would be better to make it bold and add two spaces after for a line break.

Reply:
Have you seen what the red caution box looks like in the rendered site? It looks quite "bold" already I think...

Reply:
Red box? Is it visible only on the site?

Reply:
Yes. This is a special syntax for docfx (our documentaton generation system) which generates a red box like you can see on [the current release notes](http://www.npgsql.org/doc/release-notes/4.0.html#breaking-changes-from-32).